ci/requirements/min-all-deps.yml

```diff
@@ -50,6 +50,6 @@ dependencies:
   - sparse=0.13
   - toolz=0.11
   - typing_extensions=4.0
-  - zarr=2.10
+  - zarr=2.12
```

Comment:
you can roll this back or pull in `main`.  #8022 included a broader set of version bumps.